SCClassLibrary/Common/GUI/Base/Grid.sc

```diff
@@ -483,13 +464,87 @@ ExponentialGridLines : AbstractGridLines {
 	}
 }
 
-
 BlankGridLines : AbstractGridLines {
 
-	getParams {
-		^()
+	getParams { ^() }
+
+	prCheckWarp { }
+}
+
+DrawGridTest : DrawGrid {
```

Comment:
Or is it something that could be of a more general purpose, then it would be a `DrawGridView` and may be a subclass of `View`.

Reply:
Well, the original implementation was `DrawGrid.test` so the class name is related to that, AFAIU.
However, I do get that it can be confused with unit tests so maybe another name would be better.

Reply:
Yes, "view" would only be appropriate if it were generalized to something multi-purpose. I like `DrawGridDisplay`, and the corresponding verb would work, too.

Reply:
This brings up a conceptual question about the need for the class at all, which I'll bring up in the main conversation thread.

I like "Display" as well. The method `*test` was in the original implementation. It refers to its intended use, but if we're ok with a breaking change, we could consider changing it.

Reply:
I think it would be ok to have a breaking chnage here, because it is nothing that should be used in systems, but more in exploration.